.vib/prometheus/vib-action.config

```diff
@@ -0,0 +1 @@
+verification-mode=SERIAL
```

Comment:
Could you add a comment here in the PR explaining the reason of using serial verification? Especially taking into account that it is a S4 instance

Reply:
Changed. We are setting now the `ClusteRrole` name appending the namespace, we don't need serial verification anymore.